molecule_libvirt/playbooks/templates/vm/vm.xml.j2

```diff
@@ -21,7 +21,7 @@ or other application using the libvirt API.
     <vmport state='off'/>
   </features>
   <cpu mode='custom' match='exact' check='none'>
-    <model fallback='allow'>{{ item.cpu | default('qemu64')}}</model>
+    <model fallback='allow'>{{ item.cpu_model | default('qemu64')}}</model>
```

Comment:
That seem unrelated, if is a fix it should be done as separated PR so we can easily track them.

Reply:
My bad. I have commited this change in the wrong branch.